website/docs/reference/warehouse-setups/databend-setup.md

```diff
@@ -10,7 +10,7 @@ meta:
   core_version: 'v1.0.0 and newer'
   cloud_support: Not Supported
   min_supported_version: 'n/a'
-  slack_channel_name: '#db-databend-cloud'
+  slack_channel_name: '#dbt-databend-cloud'
   slack_channel_link: 'https://datafusecloud.slack.com/archives/C04H167UABF'
```

Comment:
does this link exist? You should request a channel in the #channel-requests channel of our Slack

Reply:
Yes, this link exists. I want to create a new channel in dbt-labs slack but not got enough votes.

Reply:
hey @hantmac thanks for clarifying that! the slack channel link is different to the slack channel name so if you update the slack channel name, that can clarify things for users. let me know what you think!